modules/ai/assets/js/editor/pages/form-text/index.js

```diff
@@ -50,25 +49,19 @@ const FormText = (
 		getControlValue,
 		setControlValue,
 		additionalOptions,
-		credits: initialCredits,
+		credits,
 	},
 ) => {
 	const initialValue = getControlValue() === additionalOptions?.defaultValue ? '' : getControlValue();
-	const {
-		prompt,
-		setPrompt,
-		send,
-		isLoading,
-		error,
-		resetError,
-		result,
-		setResult,
-		sendFeedback,
-		credits,
-	} = useTextPrompt( { initialResult: initialValue, initialCredits } );
+
+	const { data, isLoading, error, reset, send, sendUsageData } = useTextPrompt( { result: initialValue, credits } );
+
+	const [ prompt, setPrompt ] = useState( '' );
 
 	const searchField = useRef( null );
 
+	const resultField = useRef( null );
```

Comment:
Why using ref and not normal state?

Reply:
I don't need it to be controlled, I just need to read its value.